templates/instantanswer/new_ia.tx

```diff
@@ -76,25 +53,29 @@
 		<div class="ia-new__title tx-clr--dk">Title</div>
         <p class="error-message hide" id="id-error">An Instant Answer with this title already exists: <a href=""></a></p>
         <p class="error-message hide" id="id-empty-error">The value for this field can't be empty</p>
-		<input class="frm__input wizard_field_insert" id="name-input" placeholder="e.g., AlternativeTo" />
+		<input class="frm__input wizard_field_insert" id="name-input" placeholder="e.g., AlternativeTo" <: if $c.session.ia_data { :>value="<: $c.session.ia_data.name :>"<: } :>/>
 	    </div>   
 	    
 	    <div class="wizard_field">
 	      <div class="ia-new__title tx-clr--dk">Description</div>
 		<p class="field_desc tx-clr--lt">Describe the purpose of your IA. This will appear on the public Instant Answer Page. </p>
-		<textarea cols="100" class="frm__text wizard_field_insert" id="description-input" placeholder="e.g., AlternativeTo lists alternatives to web-based software, desktop computer software, and mobile apps, and sorts the alternatives by various criteria, including the number of registered users who have clicked the Like button for each of them on AlternativeTo."></textarea>
-	    </div>
+		<textarea cols="100" class="frm__text wizard_field_insert" id="description-input" 
+        placeholder="e.g., AlternativeTo lists alternatives to web-based software, desktop computer software, and mobile apps, and sorts the alternatives by various criteria, including the number of registered users who have clicked the Like button for each of them on AlternativeTo."><: if $c.session.ia_data { :><: $c.session.ia_data.description :> <: } :></textarea>
+        </div>
 	 
 	    <div class="wizard_field">
 	      <div class="ia-new__title tx-clr--dk">Example Query</div>
 		<p class="field_desc tx-clr--lt">List out some example queries that will trigger this Instant Answer.</p>
-		<input class="frm__input wizard_field_insert" id="example_query-input" placeholder="e.g., Alternative To Notepad, Alternative To Spotify for Windows" />
+		<input class="frm__input wizard_field_insert" id="example_query-input"  
+        <: if $c.session.ia_data { :>value="<: $c.session.ia_data.example_query :><: if $c.session.ia_data.other_queries { :>,<: for $c.session.ia_data.other_queries -> $query { :> 
+        <:$query :>, <: } :><: } :>"<: } :> 
```

Comment:
pretty sure you can combine sequential xslate tags. `<: } :><: } :>` becomes `<: } } :>` if it must be inline